src/components/popups/CreateTask.tsx

```diff
@@ -115,6 +118,7 @@ const CreateTask = ({ isOpen, onClose }) => {
         }
       );
       //reset input fields after submitting
+      setIsLoading(false);
```

Comment:
put after catch block (otherwise might never be reset)

Reply:
Moved down